codebuild_formation.yaml

```diff
@@ -1,115 +1,83 @@
 AWSTemplateFormatVersion: 2010-09-09
-Transform: 'AWS::Serverless-2016-10-31'
+Transform: "AWS::Serverless-2016-10-31"
 
 # Adapted from: https://github.com/aws-samples/aws-codebuild-samples/blob/master/cloudformation/continuous-integration-pull-request-checks.yml
-# to create: aws cloudformation deploy --stack-name=codebuild-stdlib-utils --parameter-overrides "CodeCommitRepoName=stdlib_utils" --template-file=codebuild_formation.yaml --capabilities CAPABILITY_NAMED_IAM
 
-Description: Perform continuous integration pull request checks on a CodeCommit repository with CodeBuild
+# Note (Eli 5/3/30): you must connect your AWS account to your github account using OAuth manually before this will work (one-time). To do so, just attempt to create a CodeBuild project through the console--you don't need to finish creating it, just get to the point where you've created the OAuth connection.
+
+# to create: aws cloudformation deploy --stack-name=codebuild-stdlib-utils-publish-to-pypi --template-file=codebuild_formation.yaml --capabilities CAPABILITY_NAMED_IAM
 
-Parameters:
-  CodeCommitRepoName:
-    Description: A CodeCommit repository that contains the application code. Must be in same region as this stack.
-    Type: String
-    # Default: aws-codebuild-samples
+Description: Perform continuous integration pull request checks on a CodeCommit repository with CodeBuild
```

Comment:
can you change this in the python-github-template to say "Github" instead of "CodeCommit". and then fix it in other places over time as you create other PRs----not high priority, but we should at least change it in the template so we stop duplicating the error